scenario/3_struct_extend_pattern_unexported_in_generated.yml

```diff
@@ -0,0 +1,31 @@
+input:
+    generated/convert.go: |
+        package generated
+        // we do not allow use of unexported intToString today, even though the
```

Comment:
This feels a little overengineered to me, I'm okay with erroring on unexported/private methods, but I think all new settings related to this (AllowUnexported) can be omitted. These *can* be added, when they are needed in the future.

Setting the output_dir & output_package_name seems not really necessary in the scenario yaml.

Reply:
ack, I removed AllowUnexported and the test scenario settings. The code to check if unexported is allowed is a bit awkward (because it uses somewhat unrelated setting variable) ... but ok for now I guess.